.circleci/config.yml

```diff
@@ -32,37 +32,13 @@ jobs:
         name: Which bundler?
         command: bundle -v
 
-    # Restore bundle cache
-    - restore_cache:
-        keys:
-        - rails-demo-bundle-v2-{{ checksum "Gemfile.lock" }}
```

Comment:
You could keep the cache and just change the key name to build a new cache.

Reply:
Trying that now. I'll rebase my other PR if this works fine.